frontend/app/actions/Fallbacks.scala

```diff
@@ -16,7 +16,7 @@ object Fallbacks {
     TemporaryRedirect(controllers.routes.Outages.maintenanceMessage.absoluteURL(secure=true))
 
   def memberHome(implicit request: RequestHeader) =
-    redirectTo(controllers.routes.FrontPage.welcome)
+    redirectTo(controllers.routes.FrontPage.welcome())
```

Comment:
in scala 2.13 calling zero arg functions without using parenthasis is deprecated, so I added them in.